doc/examples/1_meshing/plot_markers.py

```diff
@@ -87,10 +91,196 @@
     ax.scatter(marker.x(), marker.y(), s=(2 - nr) * 20)
 ax.set_title('marker positions')
 fig.show()
+"""
+###############################################################################
+polygon1 = mt.createPolygon(
+    [[0.0, 0.0], [1.0, 0.0], [1.0, -1.0], [0.0, -1]],
+    isClosed=True,
+    marker=1,
+)
+
+polygon2 = mt.createPolygon(
+    [[0.25, -0.25], [0.75, -0.25], [0.75, -0.75], [0.25, -0.75]],
+    isClosed=True,
+    marker=2,
+)
+
+plc = mt.mergePLC([polygon1, polygon2])
+
+ax, cb = pg.show(plc, markers=True)
+
+fig = ax.get_figure()
+for nr, marker in enumerate(plc.regionMarker()):
+    print(
+        'Position marker number {}:'.format(nr + 1),
+        marker.x(), marker.y(), marker.z()
+    )
+    ax.scatter(marker.x(), marker.y(), s=(4 - nr) * 20, color='k')
+ax.set_title('marker positions')
+fig.savefig('polygons_test_abs.pdf')
```

Comment:
This looks great, thank you!
I am just not sure in which directory these files get written during (jenkins) build of the documentation and if they are uploaded to pygimli.org (in addition to the png in the formatted HTML). I would prefer `fig.show()` over `fig.savefig()` to avoid file output in dynamically generated examples.

Reply:
Sorry, the example needs more polishing, I know. Can I somehow disable notifications without closing the PR? I just pushing my work for today and plan to continue tomorrow...